demo/app/App_Resources/iOS/build.xcconfig

```diff
@@ -5,3 +5,4 @@
 // DEVELOPMENT_TEAM = YOUR_TEAM_ID;
 ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 ASSETCATALOG_COMPILER_LAUNCHIMAGE_NAME = LaunchImage;
+CODE_SIGN_ENTITLEMENTS = demo/Resources/demo.entitlements;
```

Comment:
@lini is this a typo here or there is a path pointing to `demo/Resources/demo.entitlements`